src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -60,11 +61,27 @@ export type CardanoSignTransactionParams = {
     auxiliaryData?: CardanoTxAuxiliaryData,
     validityIntervalStart: number,
     scriptDataHash?: string,
+    collateralInputsWithPath: InputWithPath[],
     protocolMagic: number,
     networkId: number,
     witnessPaths: Path[],
 };
 
+function transformInput(input: any) {
```

Comment:
Let's move this into a `cardanoInputs.js` helper file under `./helpers`. It is perhaps a bit of a overkill but IMHO it's better than having it here. Then it would be more consistent with the other tx elements. Also if in the end we decide to use different types for inputs and collateral inputs than we can put `transformCollateralInput` into that file also.